packages/react/src/components/Tile/Tile.js

```diff
@@ -287,6 +287,7 @@ export class SelectableTile extends Component {
           id={id}
           className={`${prefix}--tile-input`}
           value={value}
+          readOnly
```

Comment:
should we set an empty default `onChange` handler instead of forcing read only?

```suggestion
          onChange={this.props.onChange}
```

and then modifying `SelectableTile.defaultProps` and `SelectableTile.propTypes`

Reply:
For sure! Just because I don't know, why is this way preferable to adding `readOnly`?

Reply:
right now in the PR it looks like the `readOnly` value is hard coded and not modifiable so the user isn't able to remove the read only state

but I think in any case, read only checkboxes can be confusing because they are still interactable (`checked` attribute can still be modified) even though their values will not change

since this is a selectable tile component, I'm not sure we would want to set the default checkbox state to read only and have the user manually set that to falsevs just providing a change event handler

Reply:
I believe I made these changes! Let me know if anything needs to changed, especially the `SelectableTile.propTypes` part, i wasn't sure what to put there.